docs/monitoring/monitoring-metricbeat.asciidoc

```diff
@@ -0,0 +1,118 @@
+
+. Enable the collection of monitoring data in the {stack}. Set 
+`xpack.monitoring.collection.enabled` to `true` on the production cluster. +
++ 
+--
+For example, you can use the following APIs to review and change this setting:
+
+[source,js]
+----------------------------------
+GET _cluster/settings
+
+PUT _cluster/settings
+{
+  "persistent": {
+    "xpack.monitoring.collection.enabled": true
+  }
+}
+----------------------------------
+
+For more information, see {ref}/monitoring-settings.html[Monitoring settings in {es}] 
+and {ref}/cluster-update-settings.html[Cluster update settings].
+--
+
+. Disable the default collection of {kib} monitoring metrics. +
++
+--
+Add the following setting in the `kibana.yml` file on your production cluster: 
+
+[source,yaml]
+----------------------------------
+xpack.monitoring.kibana.collection.enabled: false
+----------------------------------
+
+Leave the `xpack.monitoring.enabled` set to its default value (`true`). 
+
+For more information, see 
+<<monitoring-settings-kb,Monitoring settings in {kib}>>.
+--
+
+. {metricbeat-ref}/metricbeat-installation.html[Install {metricbeat}] on the 
+same node as {kib}.
+
+. Enable the {kib} module in {metricbeat}. +
++
+--
+For example, to enable the default configuration in the `modules.d` directory, 
+run the following command:
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+metricbeat modules enable kibana
+----------------------------------------------------------------------
+
+Alternatively, you can enable module configs in the `metricbeat.yml` file. For 
+more information, see 
+{metricbeat-ref}/configuration-metricbeat.html[Specify which modules to run] and 
+{metricbeat-ref}/metricbeat-module-kibana.html[{kib} module]. 
+--
+
+. Configure the {kib} module in {metricbeat}. +
++
+--
+For example, you can change the default module configuration by modifying the 
+`kibana.yml` file in the `modules.d` directory:
+
+[source,yaml]
+----------------------------------
+- module: kibana
+  metricsets:
+    - stats
+  period: 10s
+  hosts: ["http://localhost:5601"] <1>
+  xpack.enabled: true
+----------------------------------
+<1> This setting identifies the host and port number that are used to access {kib}.  
+
+NOTE: If you configured {kib} to use <<configuring-tls,encrypted communications>>, 
+you must access it via HTTPS. For example, `https://localhost:5601`. 
+
+--
+
+. Identify where to send the monitoring data. +
++
+--
+TIP: In production environments, we strongly recommend using a separate cluster 
+(referred to as the _monitoring cluster_) to store the data. Using a separate 
+monitoring cluster prevents production cluster outages from impacting your 
+ability to access your monitoring data. It also prevents monitoring activities 
+from impacting the performance of your production cluster.
+
+For example, specify the {es} output information in the `kibana.yml` file in the 
```

Comment:
Actually this should be `... the {es} output information in the `metricbeat.yml` file:`